build-packages.cmd

```diff
@@ -1,18 +1,47 @@
 @if "%_echo%" neq "on" echo off
-setlocal
+setlocal EnableDelayedExpansion
 
 set packagesLog=build-packages.log
 echo Running build-packages.cmd %* > %packagesLog%
 
-if /I [%1] == [/?] goto Usage
+set native=false
+set options=/nologo /v:minimal /flp:v=detailed;Append;LogFile=%packagesLog%
+
+:Loop
+if [%1]==[] goto Begin
+
+if /I [%1]==[/?] goto Usage
 if /I [%1] == [/help] goto Usage
 
+if /I [%1] == [/n] (
+  set native=true
+  goto Next
+)
+
+if /I [%1] == [/native] (
+  set native=true
+  goto Next
+)
+
+echo Unrecognized argument '%1'
```

Comment:
The nice thing about having the command-line arguments to these scripts is discoverability.  It's much easier to understand that "/native" means publish native packages and I can discover it from usage or cracking open the script I'm trying to use, than it is to track down the property name "SkipMangedPackageBuild" when I'm trying to do that.  If this is not a common scenario, then it doesn't really need to be part of this script (per Wes' suggestion).